javascript-if/main.js

```diff
@@ -0,0 +1,65 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(num) {
+  if (num < 5) {
+    return true;
+  }
+  return false;
+}
+
+function isEven(num) {
+  return !(num % 2);
+}
+
+function startsWithJ(str) {
+  if (str[0] === 'J') {
+    return true;
+  }
+  return false;
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  }
+  return false;
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  }
+  return false;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH >= 0 && pH < 7) {
+    return 'acid';
+  }
+  if (pH > 7 && pH <= 14) {
+    return 'base';
+  }
+  return 'neutral';
+}
```

Comment:
This should also properly handle invalid pH levels:

---

![image](https://user-images.githubusercontent.com/16143771/137215859-f064bbde-2701-4df2-996b-f9090f0f786d.png)